dist/index.html

```diff
@@ -2,7 +2,7 @@
 <html>
   <head>
     <meta charset='UTF-8'>
-    <title>Trisha & Kayla's FitLit</title>
+    <title>Demaceo, Peyton and Jeff's FitLit</title>
```

Comment:
Doesn't look like too much was changed here

Reply:
it is now **OUR** FitLit site